core/nodejsActionBase/app.js

```diff
@@ -15,74 +15,81 @@
  * limitations under the License.
  */
 
+// __OW_ALLOW_CONCURRENT: see docs/concurrency.md
 var config = {
         'port': 8080,
         'apiHost': process.env.__OW_API_HOST,
-        'allowConcurrent': process.env.__OW_ALLOW_CONCURRENT
+        'allowConcurrent': process.env.__OW_ALLOW_CONCURRENT,
+        'requestBodyLimit': "48mb"
```

Comment:
given than OW has a limit that is checked earlier, would it be possible for Knative to check this limit earlier too, so it doesn't have to be maintained in each container ?

Reply:
good point - in fact why would knative care at all?

Reply:
In general, we were just trying to make any config. settings avail. to the downstream impls. (in case they needed the value or wished to adjust).  Agree that it can be removed if it has no value at this time beyond the initial (common) configuration in app.js.